WooCommerce/Classes/System/DefaultFeatureFlagService.swift

```diff
@@ -3,8 +3,6 @@ struct DefaultFeatureFlagService: FeatureFlagService {
         switch featureFlag {
         case .productList:
             return BuildConfiguration.current == .localDeveloper
-        case .stats:
-            return BuildConfiguration.current == .localDeveloper
```

Comment:
I was wondering if we could set it true here for the first few releases, in case we have an issue (hopefully not but in case) that we can quickly turn the feature off?

Reply:
Oooh yes, good idea! I'll go back and revert these changes, then make the feature flag set to true. This is my first time "removing" a feature flag, so I'm going to wait for a second review before merging into the 2.7 branch.